### ISSUE TITLE
Self-hosted Plex plays only first 10% when the media played is being transcoded

### ISSUE THREAD

**Issue by northys** [safing/spn#38]
**Which Website or Server are you having troubles with?**

Self-hosted plex (`https://www.plex.tv/`) gets stuck at 10% while trying to play media that has to be transcoded.

**What did not work?**

Plex works over local network. It uses their DNS to spy on users though and maybe this is the problem. Portmaster itself won't break this though. Only when I'm connected to SPN and play media that is being transcoded (simple change from mkv to mp4 container is enough) only first 10% of media is played. When I disconnect from SPN and reload the plex website it works like a charm (while portmaster is still running).

I can't see any troubles in network log though. It looks the same to me when I look at it. I'll try to investigate this and I'll keep you updated.

**Comment by dhaavi** [comment]
This is really weird. Are connecting to plex using a LAN IP address?

**Comment by northys** [comment]
yes, through their spying DNS `192-168-1-3.ThisMustBe32CharactersLongAlfanu.plex.direct` that resolves to anything that is in the first part. most of the requests in network log (in browser) passes with 200. when I disable SPN it doesn't look to me like there is more / less 200 / failed requests. but the count is different. something must be failing and i hope i'll figure it out.

```
192-168-1-3.ThisMustBe32CharactersLongAlfanu.plex.direct. 17 IN	A 192.168.1.3
```

**Comment by northys** [comment]
in portmaster UI everything looks green as well. 10.10.0.3 is the same server IP @ private wireguard VPN, 192.168.1.33 is the same server IP but on wireless connection. so it just checks if it's available but prefers LAN (192.168.1.3)

![Screenshot from 2022-01-11 01-36-31](https://user-images.githubusercontent.com/2878126/148861021-328dc2b4-5494-4316-a2d7-290e3504744d.png)

that's how I have networking configured for plex (available for plex pass (premium) users only)
![Screenshot from 2022-01-11 01-39-46](https://user-images.githubusercontent.com/2878126/148861270-b3d8b094-08b3-430b-945b-10e2ecc7ef60.png)
![Screenshot from 2022-01-11 01-39-40](https://user-images.githubusercontent.com/2878126/148861272-4f3b7dd9-157f-4d84-b504-1b85c1f1e49e.png)

**Comment by northys** [comment]
@dhaavi I tried this without SPN while I was connected outside my home metwork at work and I realized this is not SPN related problem. Closing...
